src/Particular.CodeRules/Particular.CodeRules.csproj

```diff
@@ -16,6 +16,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.2.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.0.0" PrivateAssets="contentfiles" />
```

Comment:
The default value of `PrivateAssets` is `"contentfiles;analyzers;build"`, and that results in the following in the nuspec file of the `Particular.CodeRules` package:

```xml
<dependency id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="3.0.0" exclude="Build,Analyzers" />
```

With `PrivateAssets="contentfiles"`, this changes to:

```xml
<dependency id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="3.0.0" />
```

and that matches the way that `Microsoft.CodeAnalysis.FxCopAnalyzers` depends on other analyzers, to bring them into consuming projects as transitive dependencies:

```xml
<dependency id="Microsoft.CodeQuality.Analyzers" version="[3.0.0, 3.0.0]" />
<dependency id="Microsoft.NetCore.Analyzers" version="[3.0.0, 3.0.0]" />
<dependency id="Microsoft.NetFramework.Analyzers" version="[3.0.0, 3.0.0]" />
<dependency id="Microsoft.CodeAnalysis.VersionCheckAnalyzer" version="[3.0.0, 3.0.0]" />
```

Reply:
Unless you have a specific reason why you'd want to prevent content files from also being transitively referenced, I suggest using `PrivateAssets="none"` instead. That's what I do for the transitive references in [Particular.Packaging](https://github.com/Particular/Particular.Packaging/blob/master/src/Particular.Packaging/Particular.Packaging.csproj#L16-L17)

Reply:
@bording that's what I did first, but that resulted in this in the nuspec:

```xml
<dependency id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="3.0.0" include="All" />
```

Whereas I wanted to match the way FxCop was bringing in transitive analyzers. However, it turns out that `include="All"` is the [default](https://docs.microsoft.com/en-us/nuget/reference/nuspec#dependencies-element) anyway, and I think it's better to be explicit, since we do want to bring in the entirety of FxCop. It seems that something is off in the SDK around this stuff. We've made the change you suggested. Thanks for calling it out. 👍